tools/inflector/inflector.go

```diff
@@ -31,16 +30,14 @@ func Sentenize(str string) string {
 	if str == "" {
 		return ""
 	}
+	str = UcFirst(str)
```

Comment:
(nitpick) please add a newline before the assignment to make it more clear visual that is after the if and return block.